academy/0.0-B9lab-Blockchains/4_consensus.md

```diff
@@ -0,0 +1,112 @@
+
+If a node wishes to distort the ledger in a PoW system, it must first acquire an **authoritative position**, otherwise it will be unable to influence the ledger. Acquiring an authoritative position implies overcoming the *combined* problem-solving capacity of the network and maintaining that lead over time. This known attack vector is called the 51%-attack. As the name suggests: if a single party acquires more than 50% of the total problem-solving capacity of the network, that party is theoretically able to alter the consensus. 
+
+The control mechanism for the amount of work is called **difficulty** and it guarantees a given average block creation time. PoW networks set a target average time for a solution to be found by *any* node on the network. Difficulty adjusts to compensate for increasing/decreasing total network problem-solving capacity. Thus, PoW networks do not get faster as more compute capacity is added. They become more resilient by increasing difficulty, which raises the threshold a 51% attacker will need to overcome.
+
+## Proof-of-Stake (PoS)
+
+<!-- TODO ![PoS](images/pos.png) -->
+
+**Proof-of-stake (PoS)** is another method of selecting the authoritative node for a given block. PoS is based on the assumption that those with the most to lose are the most incentivised to safeguard network integrity.
+
+A successful proof-of-stake system must address the problem of "nothing at stake". That is, randomly-selected validators must face a disincentive for bad behaviour as well as a high probability that bad behaviour will be detected. The burden of detection usually falls on the rest of the network that can either accept or reject the validator's opinion. A solution to the disincentive aspect is to extract a penalty for emitting opinions that are ultimately rejected by the network; validators face economic penalties when they generate blocks that are rejected by sizable numbers of other nodes. A validator is thus incentivised to generate blocks that are likely to be accepted by the network and faces economic punishment when it fails to do so.
+
+Validators place funds at risk (i.e. **the stake**). For any given block, a validator is selected in pseudo-random fashion with a validator with a larger stake having a higher probability of being selected to generate a block. While PoS systems generally reward validators with new coins for honest behavior (i.e. block rewards), validators also receive transaction fees in return for generating blocks the rest of the network accepts. 
+
+## Delegated-Proof-of-Stake (DPoS)
+
+An extension of proof-of-stake algorithms is called delegated-proof-of-stake (DPoS). It is used for example in BitShares, EOS, Steem, Lisk and Tron.
+
+<HighlightBox type="info">
```

Comment:
Deleted this highlight box because I would either include a short introduction to all projects mentioned in the previous sentence or none.